package.json

```diff
@@ -1,25 +1,25 @@
 {
   "name": "wurd",
-  "version": "6.3.0",
+  "version": "6.4.0",
   "description": "Wurd CMS client for Node",
   "repository": {
     "type": "git",
     "url": "https://github.com/wurdcms/wurd-node.git"
   },
   "main": "src/index.js",
   "dependencies": {
-    "cache-manager": "^2.9.0",
+    "cache-manager": "^2.10.0",
     "get-property-value": "^2.0.0",
-    "node-fetch": "^2.1.2",
-    "marked": "^0.4.0"
+    "marked": "^0.7.0",
```

Comment:
Hope this doesn't break some existing markdown content?

Reply:
Me too! It's actually at v0.8.0 now; it should be ok for properly formatted markdown.  The changes I noticed in other instances were where headings did not have a space after the `#` markers